### ISSUE TITLE
10th bit in the dataframe

### ISSUE THREAD

**Issue by OndrejBakan** [aquaticus/nexus433#24]
Hello there,

first of all thank you for your work! I am tinkering in Universal Radio Hacker and I think I found the meaning of the 10th bit in Nexus protocol. When you push the button on the sensor and force the update, this bit is 1. Otherwise (automatic periodic broadcast) the bit is 0.

Tested on the Auriol WEATHER STATION 4-LD5370 (from Lidl).

**Comment by aquaticus** [comment]
This feature probably makes pairing more reliable. Not sure if this is the
"standard" function, I'd bet it's rather unique for Auriol station (but
this is just guessing). I never saw an extra button before.
Thanks for information. If this makes a problem in your case just mask the
bit. Maybe it's worth implementing this in the main branch anway.

**Comment by OndrejBakan** [comment]
I saw the button in some other sensors too (Digoo DG-R8S for example). There's actually button for TX, button to switch °C/°F on the sensor's display and a 3 position switch to select the channel. It's usually in the battery compartment.

Nope, no problem for me. Actually, I am not even using your project (yet), but I found it very helpful while trying to decode the temperature by myself (my goal is to use Arduino / ESP32 to read the data).
